spec/views/collections/_show_descriptions.html.erb_spec.rb

```diff
@@ -13,6 +13,7 @@
     end
     before do
       assign(:presenter, presenter)
+      allow(Sufia::CollectionSizeService).to receive(:run).and_return(123456678)
```

Comment:
The test was checking to be certain the value was returned and show, so it is not just to keep it from blowing up but also to give the correct information to the view.